index.js

```diff
@@ -0,0 +1,213 @@
+//initialize values
+let page = 1;
+let pageSize = 20;
+let searchText = '';
+let category = 'General';  
+let topic = 'Politics';
+let country = 'gb';
+let modeLatestNews = false;
+let toggleNumber;
+let APIQuery = `https://newsapi.org/v2/everything?language=en&page=${page}&pageSize=${pageSize}&q=${topic}&apiKey=4fede2e79736471581f9259c131f0fc2`;
+    
+//grab HTML elements
+const articles = document.querySelector('.articles');
+const pageNextButton = document.querySelector('.next');
+const pagePrevButton = document.querySelector('.previous');
+const form = document.querySelector('.form');
+const textArea = document.querySelector('.search');
+const sidenavCategory = document.querySelectorAll('.category');
+const toggle = document.querySelector('#container');
+const toggleContainer = document.querySelector('#toggle-container');
+const pageTitle = document.querySelector('.page-title');
+const logo = document.querySelector('.logo');
+
+//initial API query & page render
+fetchArticles();
+
+//set 1m interval refresh
+setInterval(fetchArticles,60000);
+
+//Listener for side navigation categories
+sidenavCategory.forEach(link => {
+    link.addEventListener('click', event => {
+        modeLatestNews = true;
```

Comment:
It would be good to extract the logic for event handler to own function

Reply:
extracted all event handler logic to their own functions. thx